lib/jsonapi/include_directive.rb

```diff
@@ -12,6 +12,8 @@ module JSONAPI
   # @example 'posts.**' # => Include related posts, and all the included
   #   posts' related resources, and their related resources, recursively.
   class IncludeDirective
+    attr_reader :options, :hash
```

Comment:
unfortunately I need it in #merge! to do `merge_result.hash` (just renamed from `new`)